dotcom-rendering/src/palette.ts

```diff
@@ -1016,6 +1016,91 @@ const headingLineDark: PaletteFunction = (format: ArticleFormat) => {
 	}
 };
 
+const subheadingTextLight = ({ design, display, theme }: ArticleFormat) => {
+	switch (design) {
+		// Authoritative clear
+		case ArticleDesign.Obituary:
+			switch (display) {
+				case ArticleDisplay.Immersive:
+				case ArticleDisplay.Showcase:
+				case ArticleDisplay.Standard:
+					return sourcePalette.neutral[7];
+
+				default:
+					return 'unset';
```

Comment:
Do we not know what these should be for `NumberedList`? Would we able to just assume they're the same as `Standard` or `Showcase`?

Reply:
> I've treated `NumberedList` as the same as `Standard` and `Showcase` for now, will clarify this with Design team

Nice, thanks!